src/landscape.cpp

```diff
@@ -41,6 +41,7 @@
 #include "table/sprites.h"
 
 #include "safeguards.h"
+#include <pathfinder/water_regions.h>
```

Comment:
This include shouldn't be here (should be before safeguards), and shouldnt be `<>`?

Reply:
Automatically added by VS, should have paid more attention ;)